disvae/losses.py

```diff
@@ -204,27 +212,27 @@ def __call__(self, data, model, optimizer, is_train, storer):
         batch_size = data.size(dim=0)
         half_batch_size = batch_size // 2
         data = data.split(half_batch_size)
-        data1 = data[0].to(self.device)
-        data2 = data[1].to(self.device)
+        data1 = data[0]
+        data2 = data[1]
```

Comment:
Maybe it is better to name ```data1``` and ```data2``` something more descriptive?

Reply:
Happy to do so, but its what to call them. `data1` trains the vae and the discriminator. `data2` just trains the discriminator. Any suggestions?

Reply:
OK that is fine if they're officially called that in the paper. :+1: